relayer/relayer/src/handlers/azero.rs

```diff
@@ -170,9 +173,7 @@ impl AlephZeroEventHandler {
                     // Dry-run the tx to check for potential reverts.
                     call.clone().gas(config.eth_gas_limit).call().await?;
 
-                    info!(
-                        "Sending tx for request 0x{request_hash_hex} to the Ethereum network and waiting for {eth_tx_min_confirmations} confirmations."
-                    );
+                    info!("Sending tx for request 0x{request_hash_hex} to the Ethereum network and waiting for {eth_tx_min_confirmations} confirmations.");
```

Comment:
Maybe we can rename all `Ethereum` in logs to `EVM chain` to have the same name for both bridges?

Reply:
I kinda dont want to do it, at least not yet :P